src/main/java/it/pagopa/pn/f24/middleware/queue/consumer/service/PreparePdfEventService.java

```diff
@@ -141,22 +142,29 @@ private Mono<PreparePdfLists> buildF24FilesFromFileKeys(Map<String, F24MetadataR
                     return f24FileCacheDao.getItem(f24Request.getSetId(), cost, pathTokensInString)
```

Comment:
visti i problemi di concorrenza, potrebbe valer la pena far la getItem consistentRead?O è irrilevante per il problema che è successo?

Reply:
Per il problema in oggetto è irrilevante, la causa dell'errore non era la query in se, ma la gestione del dato ottenuto.
In linea generale sebbene non dovrebbe creare alcun problema, non vedo particolare necessità di implementare la consistent read, poichè i record dei file non sono soggetti a tante modifiche e si presuppone che nel flusso di prepare non ci siano particolari sovrapposizioni di altri utilizzi del dato.

Reply:
ok se dall'analisi per voi non c'è problema, per me ok.
Mi ero "preoccupato" perchè vedendo che poi controllava lo stato di quel file, se per qualche motivo arrivavano 2 risultati in contemporanea, magari entrambi leggono senza consistent e la procedura non si chiude in attesa di un evento che non arriverà più.